pkg/controller/actuator.go

```diff
@@ -380,10 +314,18 @@ func (a *actuator) createSeedResources(
 		return err
 	}
 
+	vpnEnvoyFilterSpec, err := envoyfilters.BuildVPNEnvoyFilterSpecForHelmChart(
+		cluster, spec.Rule, append(alwaysAllowedCIDRs, shootSpecificCIRDs...), istioLabels,
```

Comment:
`shootSpecificCIRDs` is already contained in `alwaysAllowedCIDRs`
```suggestion
		cluster, spec.Rule, alwaysAllowedCIDRs, istioLabels,
```